doc/guide/https.xml

```diff
@@ -69,7 +69,7 @@ MIIEvgIBADANBgkqhkiG9w0BAQEFAASCBKgwggSkAgEAAoIBAQCyOJ5garOYw0sm
     <para>To check which certificate <code>cockpit-ws</code> will use run
       the following command.</para>
 <programlisting>
-$ sudo remotectl certificate
+$ sudo /usr/libexec/cockpit-certificate-ensure --check
```

Comment:
This isn't correct for Debian, it will be in /usr/lib/cockpit/ instead. In general, users shouldn't call stuff from these private directories. We may need to move that to /usr/sbin/?